index.html

```diff
@@ -47,6 +45,186 @@ <h5>LET'S CONNECT</h5>
         </ul>
       </div>
     </section>
+<!-- NEW FROM HERE -->    
+    <section class="work-grid-container">
+      <section class="work1">
+        <img src="./images/Snapshoot.png" alt="Topic Snapshoot">
+        <div id="block1">
+          <h2>Tonic</h2>
+          <div id="work1frame">
+            <ul>
+              <li><div id="work1text1"><h5>CANOPY</h5></div></li>
+              <li><div id="work1counter"></div></li>
+              <li><div id="work1role"><h5>Back End Dev</h5></div></li>
+              <li><div id="work1counter1"></div></li>
+              <li><div id="work1text"><h5>2015</h5></div></li>
+            </ul>
+          </div>
+          <div class="work1primary-text">
+            <p>A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.</p>
+          </div>
+          <div id="work1tags">
+            <ul>
+              <li><div id="work1tag1"><div id="work1html1">html</div></div></li>
+              <li><div id="work1tag2"><div id="work1css1">css</div></div></li>
+              <li><div id="work1tag3"><div id="work1javascript1">javascript</div></div></li>
+            </ul>                        
+          </div>
+          <div class="work1Actions">
+            <button type="button" id="work1see-project">
+              <strong>See Project</strong>
+            </button>
+          </div>
+        </div>
+      </section>
+      <section class="work2">
+        <img src="./images/SnapshootPortfolio.png" alt="Stories Snapshoot">
+        <div id="block2">
+          <h2>Multi-Post Stories</h2>
+          <div id="work2frame">
+            <ul>
+              <li><div id="work2text1"><h5>CANOPY</h5></div></li>
+              <li><div id="work2counter"></div></li>
+              <li><div id="work2role"><h5>Back End Dev</h5></div></li>
+              <li><div id="work2counter1"></div></li>
+              <li><div id="work2text"><h5>2015</h5></div></li>
+            </ul>
+          </div>
+          <div class="work2primary-text">
+            <p>A daily selection of privately personalized reads; no accounts or 
+            sign-ups required.</p>
+          </div>
+          <div id="work2tags">
+            <ul>
+              <li><div id="work2tag1"><div id="html2">html</div></div></li>
+              <li><div id="work2tag2"><div id="css2">css</div></div></li>
+              <li><div id="work2tag3"><div id="javascript2">javascript</div></div></li>
+            </ul>
+          </div>
+          <div class="work2Actions">
+              <button type="button" id="work2see-project">
+                <strong>See Project</strong>
+              </button>
```

Comment:
Kindly maintain the required space between the technologies listing and the button just as it appears in your chosen template. Check the below comment for more on why you should stick to the given design.

Your design:
![image](https://user-images.githubusercontent.com/20554583/185724439-f2fc72e9-11d6-4ba8-8e7b-9a5a0a410816.png)


Figma design:
![image](https://user-images.githubusercontent.com/20554583/185724430-dc28a029-e708-46bb-afea-a4c96ee0a8e1.png)

Reply:
All reviewer's requested changes added:
- As a [best practice](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md), kindly avoid the repetition of styles. This will make your code more concise. Since every project is similar, uniform classes can serve to style similar portions of the markup.

- Kindly use a more relatable value for the alt attribute. This is what the user gets to see when there is an issue displaying the image.

- Please, adjust the button and its interactions to match the Figma design. In a typical workplace experience, you might be required to reproduce a given design as strictly as possible. So, sticking to the design as much as possible (e.g. font, colors, images, text, margins) will be a good habit right now.
Take note of the following:
- the color of the button text (enabled mode)
- the text color should change on hover of the button, not just on that the text
- there's missing button pressed interaction
- the comment applies to see project buttons too
- endeavor to use given colors

- Kindly maintain the required space between the technologies listing and the button just as it appears in your chosen template. Check the above comment for more on why you should stick to the given design.
- [Optional] changes were also added:
You can also consider:
making your PR description more readable. You could use bullets for itemizing.